packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -531,4 +571,29 @@ - (void)handleSavedPath:(NSString *)path {
   _arguments = nil;
 }
 
+/**
+ * Applies NSMutableArray on the FLutterResult.
+ *
+ * If the @c pathList is not nill and count of the @c pathList is
+ * greater than 0 then call FlutterResult with the @c pathList. If
+ * the @c pathList is nill or count of the @c pathList is not
+ * greater than 0 then call FlutterResult with FlutterError.
+ *
+ * @param @pathList that should be applied to FlutterResult.
+ */
+- (void)handleMultiSavedPaths:(NSMutableArray *)pathList {
```

Comment:
can we combine `handleMultiSavedPaths` and `handleSavedPath` into a single method where it handles differently based on the length of `pathList`?
Maybe just call it `handleSavedPaths`

Reply:
I think we need `handleSavedPath` to accept image/video paths as `String` for capture mode or videos. We only use `handleMultiSavedPaths` for `PHPicker` multi-image since we need to return `List`. But if you think we should refactor and return `List` from all the places then I can do it.

Reply:
Yeah, we can factor and return List from all the places (iOS only).